lib/constants.js

```diff
@@ -56,6 +56,12 @@ const PEGIN_UNREFUNDABLE_REASONS = {
   INVALID_AMOUNT: '3',
 };
 
+const PEGOUT_REJECTION_REASONS = {
```

Comment:
Just for my understanding, these are the codes found in rskj right?

Reply:
Yes. https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/co/rsk/peg/utils/RejectedPegoutReason.java#L22